.travis.yml

```diff
@@ -14,11 +14,16 @@ install:
   - pip install -q Django==$DJANGO
   - pip install -q pytz==2016.4
   - pip install -q python-social-auth==0.2.19
+  - pip install selenium
 
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y scilab
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
 
 # command to run tests
 script:
   - python manage.py test -v 2 yaksh
+  - python manage.py test -v 2 yaksh.tests.load_test
```

Comment:
Why is this not run along with the rest?  Given the name of the class, won't it automatically be picked up?  At least a comment at a relevant location for this would be important.